scripts/gen_jit_single_image.py

```diff
@@ -22,18 +25,19 @@
 tranlist = [transforms.ToTensor(), transforms.Normalize((0.5, 0.5, 0.5), (0.5, 0.5, 0.5))]
 tran = transforms.Compose(tranlist)
 img_tensor = tran(img)
-#print(img_tensor)
+if not args.cpu:
+    img_tensor = img_tensor.cuda()
+print(img_tensor.shape)
 
 # run through model
 out_tensor = model(img_tensor.unsqueeze(0))[0].detach()
 #print(out_tensor)
 #print(out_tensor.shape)
 
 # post-processing
-out_img = out_tensor.data[0].cpu().float().numpy()
+out_img = out_tensor.data.cpu().float().numpy()
+print(out_img.shape)
```

Comment:
If this print is meant to stay here, I like to be explicit about what is being printed, like
`print("out image shape: ", out_img.shape)`

Reply:
yes, commented it out + message.

Reply:
I still see this one
```
print(out_img.shape)
```

Reply:
![image](https://user-images.githubusercontent.com/15674552/108330656-256c7a80-71ce-11eb-86b1-39c2caefb960.png)

Reply:
done